thirdpartylibrary/README.md

```diff
@@ -0,0 +1,10 @@
+# Adding External Library to your Fiori UI application
+
+# Prerequisites
+
+- You’ll need an SAP Business Technology Platform (SAP BTP) account
+- You are subscribed to SAP Business Application Studio, follow this [tutorial](https://help.sap.com/products/SAP%20Business%20Application%20Studio/9d1db9835307451daa8c930fbd9ab264/6331319fd9ea4f0ea5331e21df329539.html) for more information
+- Sub-account destination `northwind` is available, please see attached `northind` configuration
+
+# Sample Project
+A [Fiori tools Freestyle UI application](./ztravelapp/README.md) with an external library `xml-js` enabled.
```

Comment:
freestyle app should be referred to as `freestyle SAPUI5` app. Also, I suggest to make the sentence a bit more descriptive. Example: "[This freestyle SAPUI5 application](./ztravelapp/README.md) demonstrates how to add an external library `xml-js` to the project"